### DISCUSSION TITLE
Unable to DeDRM PDF on Windows 10 21H4 (fixed)

### DISCUSSION THREAD

**Discussion by iljajj** [apprenticeharper/DeDRM_tools#1734] [General]
Hi, I'm having issues with deDRMing _some_ (weirdly enough) PDFs from a library using Calibre/DeDRM. The Calibre debug log is [here](https://www.dropbox.com/s/d0w2hh1a4303gqd/tmp5xx7jzvj.txt?dl=0). Any suggestions to solve this?

Thanks in advance!

**Comment by ElleKayEm** [comment]
DeDRM 7.2.1 still has issues with some pdfs.  Try calibre 4.23 and DeDRM 6.8.1.

**Reply by iljajj** [reply]
Thanks, will try.

**Reply by iljajj** [reply]
Okay, that fixed the issue. Thanks!
